### PR TITLE
#77 - Create Cookies trait

### PR DESCRIPTION
**Pull request by PiotrFedak**
It's a simple functions to test cookies. Such as:

- Creating cookie
- Delete cookie
- Check if cookie exist 
- Assert cookie value
- Assert cookie not exists

### CODE REVIEW

**Comment by PiotrFedak** [comment]
I was thinking about adding this cookie methods into Session trait or as it now, cookie as alone seperate trait. Waiting for someone opinion for that.

File: src/Features/Traits/Cookies.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+    /**
+     * @Then cookie :name should have value :value
+     * @Then the cookie :name should contain value :value
+     * @Then the cookie named :name should have the value :value
+     */
+    public function assertCookieValue(string $name, string $value): void
+    {
+        $request = $this->getContainer()->make(Request::class);
+        $cookieValue = $request->cookie($name);
```
Comment (JakubKermes) [inline src/Features/Traits/Cookies.php:35]:
```suggestion
        $cookieValue = $this->request->cookie($name);
```
Maybe we should check existing request from HttpRequest for cookies?

File: src/Features/Traits/Cookies.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    /**
+     * @Then the following cookies should be present:
+     * @Then the cookies should be set to:
+     */
+    public function assertCookiesExist(TableNode $table): void
+    {
+        $request = $this->getContainer()->make(Request::class);
+
+        foreach ($table as $row) {
+            $cookieValue = $request->cookie($row["name"]);
```
Comment (JakubKermes) [inline src/Features/Traits/Cookies.php:70]:
```suggestion
            $cookieValue = $this->request->cookie($row["name"]);
```
same as above in all cases

File: src/Features/Traits/Cookies.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+trait Cookies
+{
+    use Application;
+    use HttpRequest;
+
+    /**
+     * @When I set cookie :name with value :value
+     * @When I create a cookie named :name with value :value
+     * @When I add a cookie :name with value :value
```
Comment (krzysztofrewak) [inline src/Features/Traits/Cookies.php:19]: Maybe `@When cookie :name with value :value is set` would more consistent with other tests?

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]

File: src/Features/Traits/Cookies.php (hunk 3 above)
Comment (PiotrFedak) [reply src/Features/Traits/Cookies.php]: Done

Comment (krzysztofrewak) [reply src/Features/Traits/Cookies.php]: deleteCookie() has the same thing.

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**